polls/settings.py

```diff
@@ -100,21 +87,36 @@ def get_env(key, default=True):
 
 
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.7/howto/static-files/
+# https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
 
 
-# Logging
+# Cache
+
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'pollsapi',
+    }
+}
+
+CACHE_MIDDLEWARE_SECONDS = 10
+
 
-from django.utils.log import DEFAULT_LOGGING
+# Polls API Features
 
-LOGGING = DEFAULT_LOGGING
-# Remove the debug only filter for console logging.
-# We want to propergate to console and papertrail
-LOGGING['handlers']['console']['filters'] = []
```

Comment:
The default behaviour changed, I've added another commit with `/500` which I used to prove that exceptions are printed in console without this line.